A module cannot import itself

Steps to reproduce 

* clone this repo
* run ```npm install```
* run ```npm run```

Npm exits with code 1, output:

> λ npm start
> 
>  buble-react-rollup-starter@1.3.0 start C:\Users\John\Documents\Repos\buble-react-rollup-starter
>  npm run build:dev && npm-run-all --parallel watch browse
> 
> 
>  buble-react-rollup-starter@1.3.0 prebuild:dev C:\Users\John\Documents\Repos\buble-react-rollup-starter
>  npm run std --silent
> 
> 
>  buble-react-rollup-starter@1.3.0 build:dev C:\Users\John\Documents\Repos\buble-react-rollup-starter
>  rollup -c config/dev.js
> 
> A module cannot import itself (C:\Users\John\Documents\Repos\buble-react-rollup-starter\node_modules\rollup-plugin-node-globals\src\global.js)
> Error: A module cannot import itself (C:\Users\John\Documents\Repos\buble-react-rollup-starter\node_modules\rollup-plugin-node-globals\src\global.js)
>     at C:\Users\John\Documents\Repos\buble-react-rollup-starter\node_modules\rollup\src\Bundle.js:265:14
> Type rollup --help for help, or visit https://github.com/rollup/rollup/wiki
> 
> npm ERR! Windows_NT 10.0.10586
> npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\John\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-cli.js" "run" "build:dev"
> npm ERR! node v6.3.0
> npm ERR! npm  v3.10.5
> npm ERR! code ELIFECYCLE
> npm ERR! buble-react-rollup-starter@1.3.0 build:dev: `rollup -c config/dev.js`
> npm ERR! Exit status 1
> npm ERR!
> npm ERR! Failed at the buble-react-rollup-starter@1.3.0 build:dev script 'rollup -c config/dev.js'.
> npm ERR! Make sure you have the latest version of node.js and npm installed.
> npm ERR! If you do, this is most likely a problem with the buble-react-rollup-starter package,
> npm ERR! not with npm itself.
> npm ERR! Tell the author that this fails on your system:
> npm ERR!     rollup -c config/dev.js
> npm ERR! You can get information on how to open an issue for this project with:
> npm ERR!     npm bugs buble-react-rollup-starter
> npm ERR! Or if that isn't available, you can get their info via:
> npm ERR!     npm owner ls buble-react-rollup-starter
> npm ERR! There is likely additional logging output above.
> 
> npm ERR! Please include the following file with any support request:
> npm ERR!     C:\Users\John\Documents\Repos\buble-react-rollup-starter\npm-debug.log
> 
> npm ERR! Windows_NT 10.0.10586
> npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\John\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-cli.js" "start"
> npm ERR! node v6.3.0
> npm ERR! npm  v3.10.5
> npm ERR! code ELIFECYCLE
> npm ERR! buble-react-rollup-starter@1.3.0 start: `npm run build:dev && npm-run-all --parallel watch browse`
> npm ERR! Exit status 1
> npm ERR!
> npm ERR! Failed at the buble-react-rollup-starter@1.3.0 start script 'npm run build:dev && npm-run-all --parallel watch browse'.
> npm ERR! Make sure you have the latest version of node.js and npm installed.
> npm ERR! If you do, this is most likely a problem with the buble-react-rollup-starter package,
> npm ERR! not with npm itself.
> npm ERR! Tell the author that this fails on your system:
> npm ERR!     npm run build:dev && npm-run-all --parallel watch browse
> npm ERR! You can get information on how to open an issue for this project with:
> npm ERR!     npm bugs buble-react-rollup-starter
> npm ERR! Or if that isn't available, you can get their info via:
> npm ERR!     npm owner ls buble-react-rollup-starter
> npm ERR! There is likely additional logging output above.
> 
> npm ERR! Please include the following file with any support request:
> npm ERR!     C:\Users\John\Documents\Repos\buble-react-rollup-starter\npm-debug.log

--- a/package.json
+++ b/package.json
@@ -35,7 +35,7 @@
     "rollup": "0.33.0",
     "rollup-plugin-buble": "0.12.1",
     "rollup-plugin-commonjs": "3.1.0",
-    "rollup-plugin-node-globals": "1.0.5",
+    "rollup-plugin-node-globals": "1.0.6",
     "rollup-plugin-node-resolve": "1.7.1",
     "rollup-plugin-replace": "1.1.1",
     "rollup-plugin-uglify": "1.0.1",

